src/pages/Login/index.tsx

```diff
@@ -85,89 +78,124 @@ const LoginPage: React.FC = () => {
   /**
    * Display login form to user
    */
-  const renderForm = () => (
-    <MaxWidthBox maxWidth={'500px'} paddingLeft={'50px'} paddingRight={'50px'}>
+  const renderLoginForm = () => (
+    <Form>
+      <EmailInput
+        label={EMAIL_LABEL}
+        onEmailChanged={setEmail}
+        value={email}
+        isTight={true}
+      />
+      <PasswordInput
+        label={PASSWORD_LABEL}
+        onPasswordChanged={setPassword}
+        value={password}
+        isTight={true}
+      />
+      <div className="forgot-password-container">
+        <ForgotPasswordLinkComponent />
+      </div>
+      <div className="submit-buttons-container">
+        <Button type="button" onClick={handleSubmit}>
+          Sign in
+        </Button>
+        <Link
+          to={{
+            pathname: FrontendRoute.CREATE_ACCOUNT_PAGE,
+            state: { email, password },
+          }}
+        >
+          <Button type="button" variant={ButtonVariant.Secondary}>
+            Register
+          </Button>
+        </Link>
+      </div>
+    </Form>
+  );
+
+  return (
+    <>
       <Helmet>
         <title>Login | {HACKATHON_NAME}</title>
       </Helmet>
-      <Form>
-        <Flex
-          alignItems={'center'}
-          flexDirection={'column'}
-          p={'4rem 0rem 0rem 5.8rem'}
+
+      <div className="login-form-container">
```

Comment:
What's the convention regarding divs with bare classNames?

Reply:
Beyond hyphen-delimited none-abbreviated lower-case, there is none - since [styled-jsx](https://github.com/vercel/styled-jsx) keeps classes scoped to just this component we don't need to be worried about conflicts (if I create a rule that effects `.login-form-container` it'll only effect `.login-form-containers` in `LoginPage`)